src/transformer_ner/test_transfomer.py

```diff
@@ -8,30 +8,33 @@
 
```

Comment:
You can skip this since this is for test purposes.  But you can get an idea of how the new function has been tested from here.